src/helpers.js

```diff
@@ -42,7 +42,10 @@ const isExperimentalApisEnabled = process.argv.includes('--enable-experimental-a
 // Helper function to check the experimental APIs flag and
 // either throw an error or log a warning message
 function checkExperimentalApisFlag(fn) {
-  const featureName = fn.name.replace('_', ' ');
+  const featureName = fn.name.replace('_', ' ')
+                        // bound added when binding context
+                        .replace('bound ', '');
```

Comment:
I don't see where 'bound ' is added, do we really need this `replace` statement?

Reply:
bound is seemingly added by the `bind(this)` added in app.js, such that the name becomes `bound getVocabs` (which is needed to provide the correct context without `call`.